.github/workflows/ci.yml

```diff
@@ -109,7 +109,9 @@ jobs:
       - name: Install Cargo Make
         uses: davidB/rust-cargo-make@v1
         with:
-          version: "0.36.0"
+          version: "0.36.0"      
```

Comment:
Ran into an API rate limit on this in the CI, is there any way to get our rate limit expanded?